src/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+export const GreyTitle = styled.h1`
+  color: ${({ theme }) => theme.colors.grey80};
+  font-size: 12px;
+  font-weight: 10;
+  font-family: "Noto Sans";
+  margin: 0px;
+`;
+
+export const StyledLink = styled(Link)`
+  all: unset;
+  font-size: 10px;
```

Comment:
Please don't use hardcoded value

Reply:
The font in the figma designs does not exist, and making an entire new component similar to link just to not add a font size 10px doesnt make much sense imo

Reply:
delete if it is not needed